dmutils/flask_init.py

```diff
@@ -60,6 +62,9 @@ def init_app(
         db.init_app(application)
     if login_manager:
         login_manager.init_app(application)
+        import dmutils.session
+        if os.environ.get('DM_USE_REDIS_SESSION_TYPE'):
```

Comment:
Could we just use the existing `SESSION_TYPE` config var here?

```suggestion
        if app.config.get("SESSION_TYPE") == "redis":
```

I think the advantage of this is 1) it doesn't need to stay as an environment variable later on 2) one less thing to remember 😁

Reply:
@lfdebrux the reason I didn't use `SESSION_TYPE` here is that this allows us to do a gradual rollout of `SESSION_TYPE` to the frontend apps without breaking existing sessions across apps. 

Once this change is rolled out though, the condition could be changed as you suggest or potentially removed entirely.

Reply:
I see... why doesn't using `SESSION_TYPE` let us do a gradual rollout?

Reply:
Or rather, doesn't it amount to the same thing? If we can change `DM_USE_REDIS_SESSION_TYPE` in one go can't we change `SESSION_TYPE` in one go? Or am I missing something? 🤔

Reply:
@lfdebrux I found in testing that trying to set `SESSION_TYPE` globally e.g. using dmrunner settings required that it already exist with some value within the app config, else it was stripped out, so it was not set successfully. Using a `DM_` prefix env var is set globally though. Happy to talk this through more with you if that would help as I also found it confusing.

Reply:
Ah, I see. Well that makes sense then, let's go with this method you've already worked out!